diff --git a/config.h b/config.h
index a0d8c1f..abdf39e 100644
--- a/config.h
+++ b/config.h
@@ -5,8 +5,8 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "mono:pixelsize=12:antialias=true:autohint=true";
-static char *font2[] = { "JoyPixels:pixelsize=10:antialias=true:autohint=true" };
+static char *font = "JetBrainsMono Nerd Font:pixelsize=16:antialias=true:autohint=true";
+static char *font2[] = { "NotoSans Nerd Font:pixelsize=10:charset=8:antialias=true:autohint=true" };
 static int borderpx = 2;
 
 /*
